ckanext/ontario_theme/templates/internal/group/index.html

```diff
@@ -33,15 +32,12 @@ <h1>{{ _('Groups') }}</h1>
           novalidate>
       {% snippet 'snippets/search_form.html',
         searchbar_title=searchbar_title,
-        hidden_searchbar_title=input_text,
         form_id=form_id,
         type=type,
         query=query,
         sorting_selected=c.sort_by_selected,
         count=c.page.item_count,
         placeholder=input_text,
-        sr_input_text=input_text,
```

Comment:
There isn't a need for the screen reader variable, because the label gets read out by the screen reader. These variables cause the label to be repeated multiple times by the screen reader. It's not ideal for screen reader users.